packages/server/src/fhir/outcomes.ts

```diff
@@ -1,20 +1,28 @@
 import { getStatus, isAccepted } from '@medplum/core';
 import { OperationOutcome } from '@medplum/fhirtypes';
-import { randomUUID } from 'crypto';
 import { Response } from 'express';
 import { Result, ValidationError } from 'express-validator';
+import { randomUUID } from 'node:crypto';
 import { buildTracingExtension } from '../context';
 
 export function invalidRequest(errors: Result<ValidationError>): OperationOutcome {
   return {
     resourceType: 'OperationOutcome',
     id: randomUUID(),
-    issue: errors.array().map((error) => ({
-      severity: 'error',
-      code: 'invalid',
-      expression: getValidationErrorExpression(error),
-      details: { text: error.msg },
-    })),
+    issue: errors
+      .array()
+      .flatMap((error) => {
+        if (error.type === 'alternative') {
```

Comment:
Because of the `oneOf` syntax, we need to flat map all the errors out of the `nestedErrors` field in the case of `ValidationError` of type `alternative`, since the errors for each group are nested under one error.